batavia/core/builtins.js

```diff
@@ -1143,10 +1149,10 @@ batavia.builtins.sum = function(args, kwargs) {
         throw new batavia.builtins.TypeError('sum() expected at most 2 argument, got ' + args.length);
     }
 
-    var total = 0;
+    var total = new batavia.types.Int(0);
```

Comment:
Yes, it turns out it is (for `sum([])`, but I didn't do it correctly. Added a test for this and fixed it in another commit.